src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,27 @@
+.sidebar-container {
+    grid-area: sidebar;
+    width: 20rem;
+    min-height: calc(100vh - 14rem);
+}
+
+.sidebar-item {
+    font-size: 2rem;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    border-radius: 0px 10px 10px 0px;
```

Comment:
Parent-height and width are rem units.

But here, you are using px for border-radius.

I don't know if I am right or wrong, but read this: https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius. 